services/galley/src/Galley/API/Util.hs

```diff
@@ -446,7 +435,7 @@ fromRegisterConversation ::
   Qualified UserId ->
   RegisterConversation ->
   Galley Public.Conversation
-fromRegisterConversation (Qualified usr localDomain) MkRegisterConversation {..} = do
+fromRegisterConversation qusr MkRegisterConversation {..} = do
```

Comment:
Couldn't you reuse `conversationView` here?

Reply:
It's different types that are involved, though if I think hard enough, maybe I would be able to find a way to unify them.

Reply:
Is it possible to use the same technique as in the local implementation? Create *some* `Conversation` object, then pass it to `conversationView` to produce the various views for users.

Reply:
I was wrong: `conversationView` takes a `Data.Conversation`, and it doesn't really make sense to create such a value on the remote backend. Maybe just leave a FUTUREWORK comment to remove this duplicated logic in the future.